src/components/TagInput/TagInput.tsx

```diff
@@ -17,6 +18,7 @@ export const TagInput = forwardRef<HTMLInputElement, TagInputProps>(({
   hint,
   label,
   onTagsChange,
+  placeholder = 'Press ENTER to search for serveral terms',
   tags,
 }) => {
```

Comment:
TagInputProps merges TagInputBaseProps with InputHTMLAttributes<HTMLInputElement>
It means any attribute accepted by input tag would be accepted for the TagInput. To take advantage of it, do not use placeholder or any of InputHTMLAttributes props, instead, get remaining props with ...props and pass it directly to the input.